Tests/Realm.Tests/Sync/SessionTests.cs

```diff
@@ -876,23 +1058,23 @@ public void Session_Error_OnSessionError_Coexistence()
             });
         }
 
-        [TestCase(ProgressMode.ForCurrentlyOutstandingWork)]
-        [TestCase(ProgressMode.ReportIndefinitely)]
-        public void Session_ProgressObservable_IntegrationTests(ProgressMode mode)
+        [TestCaseSource(nameof(DoubleModeClientResetHandlers))]
+        public void SessionIntegrationTest_ProgressObservable((ProgressMode Mode, ClientResetHandlerBase Handler) tuple)
```

Comment:
Why change this test? Seems unrelated to the client reset work.

Reply:
Just to keep it consistent with the other naming convention I picked for the other integration tests for the session. Is this wrong?

Reply:
But it's changing the arguments of the test for no obvious reason. I don't see why we'd want to test different types of client reset handlers in this progress test.

Reply:
I understand what you mean now. You're right, client resets have nothing to do here. I went over it too fast. I'll revert it to what it was before.